gbeaudry321/ser401-NASA-psyche-mission
# Us306 poi widget gbeaudry

Changes:

- Added red dot in center of screen for debug purposes (setting debug to active doesn't work on my setup for some reason so I needed a focus point)
- Interacting with POI object causes POI template widget to appear
- Interacting again causes POI template widget to close

Test:

- Enter Psyche
- Notice red dot at center of screen, and gray cube to the right of starting position
- Approach cube. When close enough a yellow outline of cube will appear and "E" will be visible above cube.
- Press "E" on keyboard. POI template widget will appear above the cube.
- Press "E" on keyboard. POI template widget will disappear.

Expected Results:

- Enter Psyche
<img width="960" alt="Screenshot 2024-03-01 144116" src="https://github.com/gbeaudry321/ser401-NASA-psyche-mission/assets/116106901/b8123d4f-a0f9-4356-abaa-5aa11d05cfeb">

- Approach cube
<img width="958" alt="Screenshot 2024-03-01 144137" src="https://github.com/gbeaudry321/ser401-NASA-psyche-mission/assets/116106901/80b2fcb8-0419-4411-97ea-44432872e523">

- Press "E"
<img width="959" alt="Screenshot 2024-03-01 144159" src="https://github.com/gbeaudry321/ser401-NASA-psyche-mission/assets/116106901/61bc991e-ebe0-4e77-9fd0-66d0b1f5360f">

- Press "E"
Widget no longer visible

Next I am working on implementing the buttons to be clicked for interacting with the widget.
